cli/pkg/publish/publisher_v3.go

```diff
@@ -154,3 +156,15 @@ func (p *DirectoryPublisher) receiver(stream dsiv3.Importer_ImportClient) func()
 		}
 	}
 }
+
+func (p *DirectoryPublisher) done(ctx context.Context) func() error {
```

Comment:
For clarity I'd rename this function to `doneHandler`.

Reply:
Renamed